www/src/views/showcase/filtered-showcase.js

```diff
@@ -118,15 +119,39 @@ class FilteredShowcase extends Component {
           aggregatedCategories={aggregatedCategories}
         />
         <ContentContainer>
-          <ContentHeader>
+          <ContentHeader
+            cssOverrides={{
+              height: `6.9rem`,
+              flexDirection: `column-reverse`,
+              alignItems: `flex-start`,
+              pt: 5,
```

Comment:
I think these are ideal for theme-uis "responsive array" syntax. You can omit values for breakpoints via null. In our case we can omit for xs by giving second item of array a value of null. We want to make the value of padding-top to 0 above sm, sm represents the third item in array. 
Check this https://theme-ui.com/sx-prop/#responsive-values
```suggestion
              pt: [5, null, 0],
```